src/test/java/net/sf/jabref/logic/importer/fetcher/GvkParserTest.java

```diff
@@ -0,0 +1,71 @@
+package net.sf.jabref.logic.importer.fetcher;
+
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import net.sf.jabref.logic.bibtex.BibEntryAssert;
+import net.sf.jabref.logic.importer.fileformat.GvkParser;
+import net.sf.jabref.model.entry.BibEntry;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class GvkParserTest {
+
+    private void doTest(String xmlName, int expectedSize, List<String> resourceNames) throws Exception {
+        try (InputStream is = GvkParser.class.getResourceAsStream(xmlName)) {
+            GvkParser parser = new GvkParser();
+            List<BibEntry> entries = parser.parseEntries(is);
+            Assert.assertNotNull(entries);
+            Assert.assertEquals(expectedSize, entries.size());
+            int i = 0;
+            for (String resourceName : resourceNames) {
+                BibEntryAssert.assertEquals(GvkParser.class, resourceName, entries.get(i));
+                i++;
+            }
+        }
+    }
+
+    @Test
+    public void emptyResult() throws Exception {
+        doTest("gvk_empty_result_becaue_of_bad_query.xml", 0, Collections.emptyList());
```

Comment:
Could you rename the file to `gvk_empty_result_becaue_of_bad_query.xml`? `s` missing in `becaue`